wireapp/kalium
# fix: soft delete messages deleted by me (AR-1876)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?
 [AR-1876]
Soft delete messages those deleted by ME!
### Issues
For this specific ticket [AR-1876], when the list of messages is only contains 1 message, and we delete that one message, the flow won't work as expected [not the flow itself, the logic we implemented] and doesn't refresh the list correctly.

### Solutions

Soft delete messages those deleted by me/current client!

### Testing

#### Test Coverage (Optional)


#### How to Test

Open an empty conversation, send a message, delete that message! it should be marked as deleted!

### Notes (Optional)
Is there any other conditions that a list with one message get's empty? if there is, then the flow needs to support it! Otherwise all settled!

### Attachments (Optional)


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

[AR-1876]: https://wearezeta.atlassian.net/browse/AR-1876
